install.sh

```diff
@@ -71,8 +71,9 @@ start() {
   fi
   
   if [ `python -c 'import gyp; print gyp.__file__' 2> /dev/null` ]; then
-  	echo "You have a global gyp installed. Please run 'sudo apt-get remove gyp'"
-  	exit 100
+    echo "You have a global gyp installed. Setting up VirtualEnv without global pakages"
+    virtualenv $C9_DIR/python
+    npm config set python $C9_DIR/python/bin/python2
```

Comment:
I think that we want to change as little global config as possible.  Does that still make sense?